docs/permissions.rst

```diff
@@ -0,0 +1,407 @@
+Permissions
+###########
+
+.. _permissions:
+
+Objects
+=======
+
+There are four kinds of objects that you can have rights on:
+
+- **Buckets**
+- **Groups**
+- **Collections**
+- **Records**
+
+
+Permissions
+===========
+
+They are two kind of permissions on an object:
+
+- **read**: It means that the given user or group of users have
+  got read only access to the object
+- **write**: It means that the given user or group of users have
```

Comment:
I would actually differenciate between create and update (which seems to be both in the "write" option here).